### PR TITLE
Minor Improvements to AccountStats plugin

### PR DESCRIPTION
**Pull request by utdrmac**
* Added configurable interval for earnings reporting
* Added 'todays' earnings to report
* Replaced references to Poloniex with 'exchange'

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x] **I have read CONTRIBUTING.md**
- [ x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x ] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [ x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [ x] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [ x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by utdrmac** [comment]
not sure why all that pushbullet stuff is still present. I merged with upstream after that PR was merged months ago.

**Comment by utdrmac** [comment]
I added this interval feature because with the default of only once per day, I was never seeing the report in the botlog because I'm only keeping the last 200 JSON lines. Bittrex is a bit noiser than Polo and so unless I load the page at the 24hr mark, i'll not see them.

File: plugins/AccountStats.py [hunk 1]
```diff
@@ -78,9 +82,9 @@ def get_db_version(self):
         return self.db.execute("PRAGMA user_version").fetchone()[0]
 
     def fetch_history(self, first_time_stamp, last_time_stamp):
-        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 50000)
+        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 5000)
         loans = []
-        for loan in reversed(history):
```
Comment (utdrmac) [inline plugins/AccountStats.py]: Didn't see the need to waste the CPU in reversing this array. It gets ordered anyway within the database.

File: plugins/AccountStats.py [hunk 2]
```diff
@@ -78,9 +82,9 @@ def get_db_version(self):
         return self.db.execute("PRAGMA user_version").fetchone()[0]
 
     def fetch_history(self, first_time_stamp, last_time_stamp):
-        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 50000)
+        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 5000)
```
Comment (utdrmac) [inline plugins/AccountStats.py]: pared this down as to query more frequently with less chance of large data bursts getting blocked or not fully sent by exchange

File: plugins/AccountStats.py [hunk 3]
```diff
@@ -108,28 +112,41 @@ def get_first_timestamp(self):
         return row[0]
 
     def notify_stats(self):
-        if self.get_db_version() == 0:
-            if self.get_first_timestamp() is not None:
-                # only log an error if there are actually loans in DB
-                self.log.log_error('AccountStats DB isn\'t ready.')
+        if (self.get_db_version() == 0) and (self.get_first_timestamp() is not None):
```
Comment (utdrmac) [inline plugins/AccountStats.py]: Simple logic combine

File: plugins/AccountStats.py (hunk 1 above)
Comment (rnevet) [inline plugins/AccountStats.py]: Could be left over from development phase. Not sure.

**Comment by rnevet** [comment]
Hi @utdrmac, Have you been using AccountStats till now? I've been noticing that Yesterdays calculations seem off (low), did you notice the same? 

Otherwise the PR looks good.

**Comment by rnevet** [comment]
When I made this module I wasn't expecting anyone to try and see it in the logs, but use the Notifications. :)

File: default.cfg.example [hunk 4]
```diff
@@ -133,6 +133,9 @@ hideCoins = True
 #Plugins allow extending Bot functionality with extra features.
 #plugins = AccountStats
 
+#[ACCOUNTSTATS]
+#ReportInterval = 1800
```
Comment (rnevet) [inline default.cfg.example]: Need to update documentation as well.
https://github.com/BitBotFactory/poloniexlendingbot/blob/master/docs/configuration.rst#accountstats-plugin

**Comment by utdrmac** [comment]
I'm traveling for work at the moment so I'll have to check the "yesterday" calculations when I get back home in a week to be sure.

**Comment by utdrmac** [comment]
Will re-submit. Having git issues.

**Pull request closed without merging**